### DISCUSSION TITLE
How to Automatically Switch to the Secondary Monitor

### DISCUSSION THREAD

**Discussion by XPRAMT** [LizardByte/community#524] [Sunshine] [answered]
### Select Topic Area

Question

### Body

I am currently utilizing `DisplaySwitch.exe /extend` to initiate a virtual monitor. However, after extending the display, I need to manually press Ctrl + Shift + Alt + F12 to switch the streaming target to the virtual monitor.
If there is a way to replicate the functionality of Ctrl + Shift + Alt + F12 through a command-line?
![螢幕擷取畫面 2024-10-08 044541](https://github.com/user-attachments/assets/dc32ec72-7e14-447f-aadd-609866ab24c9)

**Comment by KINGLIFER** [comment]
Sucks that there is no help here.

**Reply by ReenigneArcher** [reply]
Are you volunteering to help?

**Reply by KINGLIFER** [reply]
I would if I knew the answer.

**Reply by ReenigneArcher** [reply, marked as answer]
I can't provide support for `DisplaySwitch.exe`, whatever that is, but Sunshine now has built in monitor automation for Windows.

https://docs.lizardbyte.dev/projects/sunshine/latest/md_docs_2configuration.html#dd_configuration_option

[a comment by an automated account is not shown]
